pennylane/devices/default_qubit_tf.py

```diff
@@ -164,6 +165,12 @@ def __init__(self, wires, *, shots=1000, analytic=True):
         # prevent using special apply method for this gate due to slowdown in TF implementation
         del self._apply_ops["CZ"]
 
+        # TODO: add support for the CRot operation
+        # Remove CRot from the supported operations so that it will be
+        # decomposed by default
+        if "CRot" in self.operations:
+            self.operations.remove("CRot")

```

Comment:
How come `CRot` doesn't work? There appears to be a `CRot` function in `tf_ops`.

I think this can be better fixed just be including `CRot` in the `parametric_ops` dictionary 😆

Reply:
Yeah, thanks for spotting that! Wiring it in and testing that it passes when using a `TF.Variable` was missing. Added these in the latest commit.